src/app/components/ResultsCount/ResultsCount.jsx

```diff
@@ -86,6 +89,7 @@ class ResultsCount extends React.Component {
 
 ResultsCount.propTypes = {
   count: PropTypes.number,
+  page: PropTypes.string,
```

Comment:
If `page` is a string, then it shouldn't go directly to do math in line 59 and line 60. Probably nicer to convert it to string later when presenting it in the HTML. I know it might be a bit picky.